pages/community/docs/getting-started/become-a-tsc-member.md

```diff
@@ -0,0 +1,103 @@
+---
+title: 'Become a tsc member'
+weight: 1
+---
+# BECOME A  TSC MEMBER!
+
+In this article, you will learn how to become a member of the AsyncAPI Technical Steering Committee (TSC). You will know how to contribute, how to become a committer(maintainer), and realize the benefits of being a TSC member.
+
+
+The following areas are covered:
+* What is the TSC?
+* Why TSC was formed?
+* Benefits of becoming a TSC member
+* How to become a TSC member?
+
+If you prefer video, watch our [YouTube video](https://www.youtube.com/watch?v=uG_aLF9Z1F0) on how to become a TSC member.
```

Comment:
```suggestion
If you prefer video, watch our [YouTube video on how to become a TSC member](https://www.youtube.com/watch?v=uG_aLF9Z1F0).
```

For accessibility, the hyperlink should be descriptive of what kind of vide this is; merely saying it's a YT vid isn't enough 😄✌🏽